.github/workflows/build.yml

```diff
@@ -7,11 +7,14 @@
 
 name: build
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: push
+  # push:
```

Comment:
Should we trigger on any branch?

Reply:
Not necessarily, but this was a fix for getting this to run. Our real goal is to build whenever a commit is pushed to a PR, but not sure how to achieve that.

Reply:
I suggest to leave the push master filter for now and change it on the branch when you want to trigger from your branch.

pull_request trigger does what you need. The synchronize event is received whenever the PR branch is updated.